packages/design-system/src/styles/components/_Autocomplete.scss

```diff
@@ -53,6 +53,14 @@ $autocomplete-list-border: $autocomplete-list-border-width $autocomplete-list-bo
 .ds-c-autocomplete__list-item--active {
   background-color: $color-primary-alt-darkest;
   color: $color-white;
+
+  /* stylelint-disable scss/media-feature-value-dollar-variable */
+  @media (-ms-high-contrast: active), (forced-colors: active) {
+    background-color: CanvasText;
+    color: Canvas;
```

Comment:
What are your thoughts on using IE-specific values until forced colors is more globally supported? I say this because I think IE won't support the new values, but Edge supports legacy IE declarations.

Reply:
That's a good call. Updating for IE compatibility now.